specification/2.0/Specification.adoc

```diff
@@ -3058,33 +3058,33 @@ G = \frac{2 \, \left| N \cdot L \right| \, \chi^{+}(H \cdot L)}{\left| N \cdot L
 
 where χ^+^(*x*) denotes the Heaviside function: 1 if *x* > 0 and 0 if *x* <= 0. See <<Heitz2014,Heitz (2014)>> for a derivation of the formulas.
 
-Introducing the visibility function
+Introducing the visibility function $\mathcal{V}$
 
 [latexmath]
 ++++
-V = \frac{G}{4 \, \left| N \cdot L \right| \, \left| N \cdot V \right|}
+\mathcal{V} = \frac{G}{4 \, \left| N \cdot L \right| \, \left| N \cdot V \right|}
 ++++
 
 simplifies the original microfacet BRDF to
 
 [latexmath]
 ++++
-\text{MicrofacetBRDF} = V D
+\text{MicrofacetBRDF} = \mathcal{V} D
 ++++
 
 with
 
 [latexmath]
 ++++
-V = \frac{\, \chi^{+}(H \cdot L)}{\left| N \cdot L\right| + \sqrt{\alpha^2 + (1 - \alpha^2) (N \cdot L)^2}} \frac{\, \chi^{+}(H \cdot V)}{\left| N \cdot V \right| + \sqrt{\alpha^2 + (1 - \alpha^2) (N \cdot V)^2}}
+\mathcal{V} = \frac{\, \chi^{+}(H \cdot L)}{\left| N \cdot L\right| + \sqrt{\alpha^2 + (1 - \alpha^2) (N \cdot L)^2}} \frac{\, \chi^{+}(H \cdot V)}{\left| N \cdot V \right| + \sqrt{\alpha^2 + (1 - \alpha^2) (N \cdot V)^2}}
 ++++
 
 Thus, we have the function
 
 [source,c]
 ----
 function specular_brdf(α) {
-  return V * D
+  return Vis * D
```

Comment:
Why `Vis` and not `V`?

Reply:
To not confuse it with the view direction `V`. I was trying to establish the following convention:

| Math | Code | Description |
|-|-|-|
| $V$ | `V` | View direction |
| $\mathcal{V}$ | `Vis` | Visibility term |

I tried $\text{Vis}$ and $V^\text{Visbility}$, but it didn't look nice in the formulas.